gtk/src/auto/accel_group.rs

```diff
@@ -84,14 +87,17 @@ pub trait AccelGroupExt: 'static {
         f: F,
     ) -> SignalHandlerId;
 
+    #[doc(alias = "accel-changed")]
     fn connect_accel_changed<F: Fn(&Self, u32, gdk::ModifierType, &glib::Closure) + 'static>(
```

Comment:
@sdroege: I also added doc aliases for signals. Thought that would be consistent with your proposal for properties getter / setter (see other comment below)...

Reply:
Good idea, yes. Forgot about signals. That was another case where we diverged from the C API before already (CC @GuillaumeGomez )